src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets

```diff
@@ -45,6 +45,43 @@ Copyright (c) .NET Foundation. All rights reserved.
     <_GeneratedStaticWebAssetsDevelopmentManifest>$(IntermediateOutputPath)Microsoft.AspNetCore.StaticWebAssets.xml</_GeneratedStaticWebAssetsDevelopmentManifest>
   </PropertyGroup>
 
+  <!-- 
+    Targets used for publishing content from razor class libraries
+  -->
+
+  <!--
+    Called after ComputeFilesToPublish but before CopyFilesToPublishDirectory - this target is called when 
+    publishing the project to get a list of files to the output directory.
+  -->
+  <Target
+    Name="_StaticWebAssetsComputeFilesToPublish"
+    AfterTargets="ComputeRefAssembliesToPublish"
```

Comment:
`ComputeFilesToPublish` == `ComputeRefAssembliesToPublish` 🤔 

Which is it?

Reply:
Not sure what your question here is

Reply:
This is the same way _RazorComputeFilesToPublish does it